scripts/manage-views.py

```diff
@@ -29,26 +22,46 @@
     postgresql+pg8000://scott:tiger@localhost/mydatabase
 """)
     parser.add_argument('-d', '--dsn', help='Database DSN in which to manage views')
-    parser.add_argument('operation', help="Operation to perform (create | refresh)",
+    log_level_choices = 'NOTSET DEBUG INFO WARNING ERROR CRITICAL'.split()
```

Comment:
Cool, that's nice to split the script and db engine log levels to be separately configurable.